docs/dapps/sdk/signer-provider.md

```diff
@@ -0,0 +1,143 @@
+---
+sidebar_position: 30
+title: SignerProvider
+sidebar_label: SignerProvider
+---
+
+The `SignerProvider` is an abstraction of an Alephium account. It can be used to sign messages and transactions, as well as send signed transactions to the Alephium blockchain.
+
+## Creating SignerProvider
```

Comment:
We are converging to use simple verbs in title all over the documentation

e.g. "Create SignerProvider" instead of "Creating SignerProvider"